src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2381,6 +2382,17 @@ public function setIncrementing($value)
     }
 
     /**
+     * Transform each attribute in the model using a callback.
+     *
+     * @param  callable  $callback
+     * @return \Illuminate\Support\Fluent
+     */
+    public function transform(callable $callback)
+    {
+        return (new Fluent($this->toArray()))->transform($callback);
```

Comment:
I wonder why is `Fluent` even needs to be used?
`league/fractal` package provides quite good set of tools for transforming objects to API output. Perhaps something like that would be much better.